python/lsst/analysis/tools/tasks/injectedObjectAnalysis.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = ("InjectedObjectAnalysisConfig", "InjectedObjectAnalysisTask")
+
+from lsst.pipe.base import connectionTypes as ct
+from lsst.skymap import BaseSkyMap
+
+from ..interfaces import AnalysisBaseConfig, AnalysisBaseConnections, AnalysisPipelineTask
+
+
+class InjectedObjectAnalysisConnections(
```

Comment:
I do think that consolidating this with objectTableTractAnalysis would be better unless there is a compelling reason not to?

Reply:
I also lean towards consolidation, but if they ever needed to diverge in the future, we'd have to add the class in again, editing every pipeline where it's used. Otherwise, the only benefit is having different defaults. 

In retrospect we probably should have called `ObjectTableTractAnalysis` `TractTableAnalysis` since it's only the defaults that point to objectTable_tract.